datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/check/{check_name}/datadog_checks/{check_name}/check.py

```diff
@@ -5,7 +5,48 @@
 
 
 class {check_class}(AgentCheck):
+    def __init__(self, name, init_config, instances):
+        # super({check_name}, self).__init__(name, init_config, instances)
+
+        # If the check is going to perform SQL queries you should define a query manager here.
+        # More info at
+        # https://datadoghq.dev/integrations-core/base/databases/#datadog_checks.base.utils.db.core.QueryManager
+        # sample_query = {{
+        #     "name": "sample",
+        #     "query": "SELECT * FROM sample_table",
+        #     "columns": [
+        #         {{"name": "metric", "type": "gauge"}}
+        #     ],
+        # }}
+        # self._query_manager = QueryManager(self, self.execute_query, queries=[sample_query])
+        # self.check_initializations.append(self._query_manager.compile_queries)
+        pass
+
     def check(self, _):
         # type: (Any) -> None
         # Use self.instance to read the check configuration
+
+        # The following are useful bits of code to help new users started.
+
+        # Perform HTTP Requests with our HTTP wrapper.
+        # More info at https://datadoghq.dev/integrations-core/base/http/
+        # self.http.get("<url>")
```

Comment:
Could expand on this and also demonstrate sending critical service checks, though this might be too verbose:

```python
        try:
            response = self.http.get(url)
            response.raise_for_status()
            response_json = response.json()

        except Timeout as e:
            self.service_check({check_name}.can_connect, AgentCheck.OK, message="Request timeout: {}, {}".format(url, e))
            raise

        except (HTTPError, InvalidURL, ConnectionError) as e:
            self.service_check({check_name}.can_connect, AgentCheck.OK, message="Request failed: {}, {}".format(url, e))
            raise

        except JSONDecodeError as e:
            self.service_check({check_name}.can_connect, AgentCheck.OK, message="JSON Parse failed: {}, {}".format(url, e))
            raise

        except ValueError as e:
            self.service_check({check_name}.can_connect, AgentCheck.OK, message=str(e))
            raise
```

Reply:
I think this is a good idea